docs/running/tardis_widgets/generating_widgets.ipynb

```diff
@@ -4,18 +4,24 @@
    "cell_type": "markdown",
```

Comment:
<p>"TARDIS widgets that allows..."</p><p> </p><p>"This notebook represents a quick-start tutorial, but more detail on each widget (and its features) is giving in the Using TARDIS Widgets section of the documentation"</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/tardis-sn/tardis/pull/1280/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>